Form-Controls/styles.css

```diff
@@ -0,0 +1,47 @@
+
+h1{
+    text-align: center;
+    color: #354f52;
+}
+
+h2{ 
+    text-align: center;
+    color: #354f52;
+    font-size: 12px;
+}
+
+div{
+    margin-bottom: 20px;
+}
+
+.product-form{
+    background-color: #588157;
+    margin: 0px 10em;
+    padding: 20px;
+    min-width: 300px;
+}
+
+label{
+    padding:20px;
+}
+
+input{
+    margin: 10px 20px;
+    padding: 10px 30px;
+    border: 2px solid #dad7cd;
+    border-radius: 5px;
+}
+
+input[type="submit"]{
+   background-color: #344e41;
+   color: aliceblue;
+   padding: 10px 20px;
+}
+
+#size{
```

Comment:
Do you need an id here? How did you decide to use an id? Thinking through this decision carefully is useful. You might like this article to help you deliberate. https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/